data-prepper-plugins/csv-processor/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'java'
+}
+
+group 'com.amazon.dataprepper.plugins.processor.csv'
+version '2.0.0-SNAPSHOT'
+
+dependencies {
+    implementation project(':data-prepper-api')
+    implementation project(':data-prepper-plugins:common')
+    implementation 'com.fasterxml.jackson.core:jackson-databind'
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-csv'
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
+
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.2'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.2'
```

Comment:
Same as above, not required. I think these will be used used from parent project